packages/svelte/tests/runtime-runes/samples/anonymous-declarations/main.svelte

```diff
@@ -0,0 +1,4 @@
+<script>
+	import broken from './function.svelte.js';
+	import broken2 from './class.svelte.js';
```

Comment:
Names are a bit confusing: these imports are not broken.
In general, it's probably better to move this test into the validation test suite, where you can more clearly test "I expect this not to fail"